src/Raven.Server/Documents/PeriodicBackup/BackupTask.cs

```diff
@@ -264,7 +278,7 @@ public BackupResult RunPeriodicBackup(Action<IOperationProgress> onProgress, ref
                         runningBackupStatus.Version = ++_previousBackupStatus.Version;
                         // save the backup status
                         AddInfo("Saving backup status");
-                        SaveBackupStatus(runningBackupStatus, _database, _logger, _backupResult);
+                        SaveBackupStatus(runningBackupStatus, _database, _logger, _backupResult, _forTestingPurposes);
```

Comment:
is that possible to avoid passing `_forTestingPurposes`?

Reply:
Done.